core/logger/logger.go

```diff
@@ -27,7 +27,16 @@ func init() {
 	if err != nil {
 		log.Fatalf("failed to register os specific sinks %+v", err)
 	}
-	if os.Getenv("LOG_COLOR") != "true" {
+	// https://app.shortcut.com/chainlinklabs/story/33622/remove-legacy-config
+	var logColor string
+	if v1, v2 := os.Getenv("LOG_COLOR"), os.Getenv("CL_LOG_COLOR"); v1 != "" && v2 != "" {
```

Comment:
Same thought for this and the others... if we're moving away from the first format, I think it's fine to tell them they should set "CL_LOG_COLOR", but continue to quietly accept both.

Reply:
New configuration is meant to be switched to all at once, and outright rejection of old deprecated config is part of the design as a feature, so that things are not misconfigured unintentionally.

Reply:
I'm working on the follow up change to check for all legacy env vars, and one hitch is that the tests still live in both worlds and need to continue to work. I wonder if we could make one small exception here, where we allow both legacy and v2 to be used, but _only if they are exactly equal_. This way we can just set both during the transition period to support testing in CI (and for devs going back and forth). FWIW this would likely be limited to db url (plus backup) and dev mode in practice. cc @samsondav